ingestion_tools/scripts/data_validation/fixtures/data.py

```diff
@@ -18,24 +23,98 @@
 # ==================================================================================================
 
 
-def get_header(mrcfile: str) -> MrcInterpreter:
-    """Get the mrc file header for a tilt series without downloading the entire file."""
-    fs = FileSystemApi.get_fs_api(mode="s3", force_overwrite=False)
-    with fs.open(mrcfile, "rb") as f:
-        header = MrcInterpreter(iostream=f, permissive=True, header_only=True)
-    return header
+def get_header(mrcfile: str, fs: FileSystemApi) -> MrcInterpreter:
+    """Get the mrc file headers for a list of mrc files."""
+    try:
+        with fs.open(mrcfile, "rb") as f:
+            header = MrcInterpreter(iostream=f, permissive=True, header_only=True)
+        return header
+    except Exception as e:
+        pytest.fail(f"Failed to get header for {mrcfile}: {e}")
+
+
+def get_zarr_headers(zarrfile: str, fs: FileSystemApi) -> Dict[str, Dict]:
+    """Get the zattrs and zarray data for a zarr volume file."""
+    expected_subfolders = {f"{zarrfile}/{i}" for i in range(3)}.union({f"{zarrfile}/.zattrs", f"{zarrfile}/.zgroup"})
+    actual_subfolders = {"s3://" + folder for folder in fs.glob(zarrfile + "/*")}
+    if expected_subfolders != actual_subfolders:
+        pytest.fail(f"Expected zarr subfolders: {expected_subfolders}, Actual zarr subfolders: {actual_subfolders}")
+
+    fsstore = zarr.storage.FSStore(url=zarrfile, mode="r", fs=fs.s3fs, dimension_separator="/")
```

Comment:
This could use [omezarr](https://ome-zarr.readthedocs.io/en/stable/api/io.html) if possible for validating if the 3 scales exist

Reply:
After a discussion, @manasaV3 and I decided that it would be best to just keep it as simple as possible and the implementation with omezarr might be a bit clunky, so just reading the files directly is fine for now.